ci/setup_grpc.sh

```diff
@@ -7,24 +7,41 @@ set -e
 export DEBIAN_FRONTEND=noninteractive
 old_grpc_version='v1.33.2'
 new_grpc_version='v1.49.2'
+modern_grpc_version='v1.55.0'
 gcc_version_for_new_grpc='5.1'
 std_version='14'
+if [ ! -z "${CXX_STANDARD}" ]; then
+    std_version="${CXX_STANDARD}"
+fi
 install_grpc_version=${new_grpc_version}
 install_dir='/usr/local/'
 build_shared_libs=''
+build_internal_abseil_cpp=1
+GRPC_BUILD_OPTIONS=()
 usage() { echo "Usage: $0 [-v <gcc-version>] [-i <install_dir>"] 1>&2; exit 1;}
 
-while getopts ":v:i:r:s:TH" o; do
+while getopts ":v:i:mp:r:s:TH" o; do
```

Comment:
Please update the usage() helper to show all options.

Reply:
Thanks, and all options are added into usage message.